scripts/kdump.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+dst="${1:-resources}"
```

Comment:
gitignore this dir?

Reply:
Doen with commit 36b1ccf4d76edeca33eca62bd7c5c955b7315562.